Assets/SEE/Game/UI/LiveDocumentation/Buffer/LiveDocumentationBuffer.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using System.Text;
+
+namespace SEE.Game.UI.LiveDocumentation.Buffer
+{
+    /// <summary>
+    ///     LiveDocumentationBuffer contains the documentation of a class including links as chunks of
+    ///     <see cref="ILiveDocumentationBufferItem" />.
+    ///     Those hold either a simple text or a link
+    /// </summary>
+    /// <example>
+    ///     Just add Text
+    ///     <code>
+    ///         LiveDocumentationBuffer buffer = new LiveDocumentationBuffer();
+    ///         buffer.Add(new LiveDocumentationBufferText('TEXT TO ADD'));
+    ///     </code>
+    ///     <p />
+    ///     Adding Text and Links
+    ///     <code>
+    ///         LiveDocumentationBuffer buffer = new LiveDocumentationBuffer();
+    ///         buffer.Add(new LiveDocumentationBufferText('TEXT TO ADD'));
+    ///         buffer.Add(new LiveDocumentationLink("./path/to/class", "Link name"));
+    ///     </code>
+    /// </example>
+    public class LiveDocumentationBuffer
+    {
+        /// <summary>
+        ///     Constructor for the <see cref="LiveDocumentationBuffer" />
```

Comment:
Fix indentation. All comments should be indented uniformely to the rest of our code.

Reply:
I fixed it.